numpy/lib/function_base.py

```diff
@@ -3543,7 +3543,7 @@ def percentile(a, q, axis=None, out=None,
 def quantile(a, q, axis=None, out=None,
              overwrite_input=False, interpolation='linear', keepdims=False):
     """
-    Compute the `q`th quantile of the data along the specified axis.
+    Compute the qth quantile of the data along the specified axis.
```

Comment:
Alternate form "q-th", which seems pretty popular.

Reply:
change all qth and `q`th to q-th